docs/reference/extensibility/NuGet-Cross-Platform-Plugins.md

```diff
@@ -66,8 +66,10 @@ After 1 minute of inactivity a plugin is considered idle and is shut down.
 ## Plugin installation and discovery
 
 The plugins will be discovered via a convention based directory structure.
-CI/CD scenarios and power users can use an environment variable to override the behavior.
+CI/CD scenarios and power users can use environment variables to override the behavior. Note that `NUGET_NETFX_PLUGIN_PATHS` and `NUGET_NETCORE_PLUGIN_PATHS` are only available with 5.3+ version of the NuGet tooling and later.
 
+- `NUGET_NETFX_PLUGIN_PATHS` - defines the plugins that will be used by the .NET Framework based tooling (NuGet.exe/MSBuild.exe/Visual Studio). Takes precedence over `NUGET_PLUGIN_PATHS`. (NuGet version 5.3+ only)
+- `NUGET_NETCORE_PLUGIN_PATHS` - defines the plugins that will be used by the .NET Core based tooling (dotnet.exe). Takes precedence over `NUGET_PLUGIN_PATHS`. (NuGet version 5.3+ only)
 - `NUGET_PLUGIN_PATHS` - defines the plugins that will be used for that NuGet process, priority reserved. If this environment variable is set, it overrides the convention based discovery.

```

Comment:
Even though it's hinted at above, perhaps explicitly state that `NUGET_PLUGIN_PATHS` is ignored if one of the framework-specific environment variables is used?

Reply:
Makes sense. 
Thanks.